src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
```

Comment:
suggestion

          import { ApplicationConfig } from '@angular/core';

          Explantion: The import statement is correct as it is, but you might want to ensure proper error handling when using ApplicationConfig elsewhere in the application.